README.md

```diff
@@ -1,119 +1,80 @@
 
 #### Prerequisites
-Going the bare route, the following software needs to be installed:
+- Ruby: see this project's Gemfile for the version of Ruby currently being used with this project. We recommend RVM for managing multiple Ruby versions, but there are other options available
+- [Jekyll](https://jekyllrb.com/): Jekyll 3
+- [HTMLProofer](https://github.com/gjtorikian/html-proofer): used for testing links, images, and HTML. The docs site will need a passing build to be deployed, so use HTMLProofer to test everything before you push changes to GitHub.
 
-- Git: system version should be fine
-- Ruby: the version of Ruby currently being used with this project will be noted in the Gemfile. If you need to manage multiple Ruby versions, we recommend RVM, though there are similar solutions you can use.
-- [Jekyll](https://jekyllrb.com/): Jekyll version 3.
-- [HTMLProofer](https://github.com/gjtorikian/html-proofer): HTMLProofer is used for testing links, images, and HTML. The docs site needs to pass the build and tests to be deployed. You can use HTMLProofer to test things before you send changes up to GitHub.
-
-You're welcome to also use Bundler to install the Gems needed. If you are using RVM (or similar), just make sure they all play nicely together.
+You're also welcome to use Bundler to install required gems. If you are using RVM (or similar), make sure they all play nicely together.
 
 #### First Run
-To get a local copy of <circleci.com/docs/> up and running, you can run the 
-following commands:
+To get a local copy of <circleci.com/docs/>, run the following commands:
 
-```
+```bash
 git clone https://github.com/circleci/circleci-docs.git
 cd circleci-docs/jekyll
 jekyll serve
 ```
 
-Jekyll will build the site and start a web server. It can be viewed in your 
-browser at <http://localhost:4000/docs/>. To stop Jekyll and regain control 
-of your terminal, just type `CTRL-C`.
+Jekyll will build the site and start a web server, which can be viewed in your browser at <http://localhost:4000/docs/>.
 
 ####  Editing Docs
-
-All of the docs can be found in the `jekyll/_docs` directory. You can make any 
-changes that you need there, then re-run `jekyll serve` to have Jekyll rebuild 
+All docs can be found in the `jekyll/_docs` directory. Make changes there, then re-run `jekyll serve` to have Jekyll rebuild 
 and serve the site.
 
-## Jekyll Controller (JCTL)
-
+## Jekyll Controller (`jctl`)
 This is a Bash wrapper script to talk to Jekyll & Vagrant.
 
-- start: Starts Jekyll. If Vagrant isn't running, starts Vagrant as well.
-- rebuild: Tells Jekyll to rebuild the site.
-- stop: Shuts down the entire VM (`vagrant halt`), including Jekyll.
-- restart: Restarts the Vagrant machine. Basically an alias of stop then start.
+- `start`: starts Jekyll; will also start Vagrant, if not already running
+- `rebuild`: rebuilds the site
+- `stop`: shuts down entire VM (including Jekyll)
+- `restart`: `jekyll stop && jekyll start`
```

Comment:
If we're writing the command out, then it should be `./jctl stop && ./jctl start`.